src/options/views/tab-settings/index.vue

```diff
@@ -70,6 +60,21 @@
       </button>
     </div>
     <div v-show="showAdvanced">
+      <section>
+        <h3 v-text="i18n('labelGeneral')"></h3>
+        <div class="mb-1">
+          <label>
+            <span v-text="i18n('labelInjectionMode')"></span>
+            <select v-model="defaultInjectInto">
+              <option value="auto">auto</option>
+              <option value="page">page</option>
+              <option value="content">content</option>
+              <option value="wrap">wrap</option>
```

Comment:
Maybe hide `wrap` in Chrome?

Reply:
On the other hand, it may be confusing to hide, if the user transfers the backup between FF/Chrome...

Reply:
In most cases `auto` should be enough, I think we could even remove this option so that users do not have to learn a new concept.

Reply:
Hmm, what if `content` will behave internally like `content+wrap` by default and the new mode will be not `wrap` but `content (legacy)` which is internally the old `content`?

Reply:
> what if `content` will behave internally like `content+wrap` by default and the new mode will be not `wrap` but `content (legacy)` which is internally the old `content`?

Looks good, and simple.

Reply:
There is a problem, I think the keywords should be consistent either using in the settings or in `@inject-into`. But I don't think we can use `@inject-into content (legacy)`, so maybe something as a word, like `content-legacy`.

Reply:
Another problem with renaming is that it will make `content` behave differently between Chrome/FF.

These ideas would be provide a more consistent behavior:

* hiding `wrap` in Chrome
* display it as `wrap (only in Firefox)`
* add `disabled` attribute on the `<option>` element so it's unselectable

And of course a hint below the selector would be helpful.

Reply:
I'm always trying to hide the differences between browsers and provide same or similar functionality to developers and users. So,
- since `wrap` is a Firefox-only feature, I don't want to expose it to users, and not to show a different UI in Firefox.
- `content` mode is supposed to be the final way to inject scripts, just trying to get scripts executed, without ensuring same environments as in page contexts.

But I'm starting to think that, `wrap` might be better as a fallback for `page`, i.e. `page=page+wrap, content=content, auto=page+wrap+content`. It seems more consistent for both Chrome and Firefox this way.

Reply:
Yeah, that makes sense.